### PR TITLE
Update version history for 5.1.3

### PR DESCRIPTION
**Pull request by melissalinkert**
Usual caveats apply, /cc @hflynn 

I didn't mention C++ on Windows, since the state of that for release isn't yet clear.  Happy to add a "progress on ..." or whatever note if that's useful or necessary, /cc @rleigh-dundee

### CODE REVIEW

**Comment by rleigh-dundee** [comment]
@melissalinkert Assuming we merge #1873 (and it's all green now, so that should be possible), then I think we can say for the C++ side that we have implemented the CMake SuperBuild infrastructure to build the core dependencies of Bio-Formats-C++ (zlib, bzip2, png, icu, xerces, boost), and that work on porting bioformats C++ code to build on Windows is currently in progress.

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,53 @@
...
+   - Allow reading from https URLs
+   - ImageJ
+       - improved performance of displaying ROIs
+   - Command line tools
+       - fixed bfconvert to correctly create datasets with multiple files
+   - Metamorph
+       - improved detection of time series
+       - fixed .nd datasets with variable Z and T counts in each channel
+       - fixed .nd datasets that contain invalid TIFF/STK files
+       - fix dimensions when the number of planes does not match the recorded
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: fix -> fixed for consistency

File: docs/sphinx/about/whats-new.txt [hunk 2]
```diff
@@ -1,6 +1,53 @@
...
+       - fixed decompression of lossless files with multiple channels (thanks to Aaron Avery)
+   - Imspector OBF
+       - updated to support version 2 files (thanks to Bjoern Thiel)
+   - Imspector MSR
+       - improved detection of Z stacks
+   - PerkinElmer Opera Flex
+       - improved handling of multiple acquisitions of the same plate
+   - Zeiss CZI
+       - fixed error when opening single-file datasets whose names contained
+         `(`  and `)`
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: looks a bit confusing rendered, do you want to use '' quote marks you can see in formatted file instead?

**Comment by hflynn** [comment]
Two very minor quibbles but looks good, reads and formats fine.

**Comment by melissalinkert** [comment]
Fix pushed for all points.

**Comment by hflynn** [comment]
:+1: Should be good to go now

**Pull request merged**